e2e/app/page/workspace-base.ts

```diff
@@ -130,7 +130,7 @@ export default abstract class WorkspaceBase extends AuthenticatedPage {
    */
   async deleteResource(resourceName: string, resourceType: ResourceCard): Promise<string[]> {
     const resourceCard = new DataResourceCard(this.page);
-    const card = await resourceCard.findCard(resourceName, resourceType);
+    const card = await resourceCard.findCard(resourceName, resourceType, 60 * 1000);
```

Comment:
Extend timeout because the default timeout in `findCard` function is 2 seconds which is not enough.